django/db/backends/postgresql/introspection.py

```diff
@@ -71,10 +71,15 @@ def get_table_description(self, cursor, table_name):
         """
         # As cursor.description does not return reliably the nullable property,
         # we have to query the information_schema (#7783)
+        # Allow schemas (#22673)
+        table_schema = 'public'
+        parts = table_name.split('.')
+        if len(parts) > 1:
+            table_schema = parts[-2]
         cursor.execute("""
             SELECT column_name, is_nullable, column_default
             FROM information_schema.columns
-            WHERE table_name = %s""", [table_name])
+            WHERE table_schema = %s AND table_name = %s""", [table_schema, parts[-1]])
```

Comment:
Actually, additional quoting doesn't solve it all the way and e.g. the fix for #7783 that I mentioned causes problems.  And the other issue is that when creating tables, you'll have to create the schema manually first.